Readme.md

```diff
@@ -2,8 +2,13 @@
 - Install nodejs
 - Run `npm install` via terminal
 - Run `npm install cypress --save-dev` via terminal
+- Run `npm install mocha --save-dev` to install Mocha
+- Run `npm install mochawesome --save-dev` to install mochawesome
+- Run `npm install mochawesome-merge --save-dev` to install mochawesome-merge
+- Run `npm install mochawesome-report-generator --save-dev` to install mochawesome-report-generator
```

Comment:
There is no need in lines 4-8.
All necessary dependencies to install are specified in `package.json` file.
So when someone wants to use your framework, then he/she clones it and runs `npm install` and all dependencies from `package.json` file will be installed.

Reply:
Also no need to use extra words like 'via terminal'.
Example:

Install dependencies:
```
npm install
```